src/redux/store/Authentication/reducer.ts

```diff
@@ -0,0 +1,24 @@
+import { LoginStatus } from "./../../../models/LoginStatus";
+import { LOGIN, LOGOUT } from "./actionType";
+
+const initialState: { authenticated: LoginStatus } = {
+  authenticated: LoginStatus.LoggedOut,
+};
+
+const AuthenticationReducer = (state = initialState, action: any) => {
+  switch (action.type) {
+    case LOGIN:
+      const { loginCredentials } = action;
+      const authenticated =
+        loginCredentials?.password === "L#(qc{f}TaJu4%4k"
```

Comment:
I would like to have a simpler password for Devs to use, please.

Reply:
Not changed as discussed over the call